modules/govuk/manifests/apps/collections.pp

```diff
@@ -41,6 +45,7 @@
     vhost                  => $vhost,
     nagios_memory_warning  => $nagios_memory_warning,
     nagios_memory_critical => $nagios_memory_critical,
+    sentry_dsn             => $sentry_dsn,
```

Comment:
I think the way Errbit and other env vars are configured is way too tedious. I was actually hoping to avoid editing every app manifest, but that doesn't seem to be possible. In the mean time, I prefer to abstract away the setting of the env var to somewhere else. WDYT?